NtupleProducer/python/HiggsTauTauProducer_102X.py

```diff
@@ -16,7 +16,7 @@
 try: PERIOD
```

Comment:
Not necessarily for this PR, but is there any good reason to keep the legacy producers in here (102X, 94X,...)?

Reply:
Actually no, but they mean no harm

Reply:
fair enough, just thinking that at some point we could do some cleanup (again, not necessarily now)